staging/src/k8s.io/apimachinery/pkg/runtime/schema/group_version.go

```diff
@@ -51,7 +51,7 @@ func (gr GroupResource) Empty() bool {
 	return len(gr.Group) == 0 && len(gr.Resource) == 0
 }
 
-func (gr *GroupResource) String() string {
+func (gr GroupResource) String() string {
```

Comment:
Why this change? It makes this type inconsistent with the other types in this file.

Reply:
Thanks for review @liggitt. The reason is I wanted to do this: `resource.GroupResource().String()`. and  `resource.GroupResource()` doesn't return a pointer. It's weird, but I can do it without this change.

Good catch. I will change it back.

Reply:
I reverted the change. The point of using pointer as receiver is probably to avoid copying values, but the previous methods use values receivers and return values. so it's confusing. however, it's out of the scope of this change.